pages/common/webpack.md

```diff
@@ -0,0 +1,19 @@
+# webpack
+
+> Bundle your js files and other assets into a single output file.
+
+- Create a single output file from an entry point file:
+
+`webpack {{app.js}} {{bundle.js}}`
+
+- Load css files too from your js file. (This uses the css loader for .css files):
+
+`webpack {{app.js}} {{bundle.js}} --module-bind 'css=css'`
+
+- Pass a config file and show compilation progress:
+
+`webpack --config {{webpack.config.js}} --progress`
```

Comment:
The example contents is kind of pretty big. So I was not sure whether to include it or not. Here is one such example
```javascript
module.exports = {
  entry: './src/main.js',
  output: {
    path: './dist',
    filename: 'output.js'
  }
};
```
How should I include it ?